docker-compose-developer-pg.yml

```diff
@@ -0,0 +1,74 @@
+version: '3.2'
+
+services:
+
+  postgres:
```

Comment:
Another approach might be using compose profile like here 
https://github.com/grafana/oncall/blob/dev/docker-compose.yml#L173

But I'm fine with both approaches

Reply:
I was looking at the `profile` option, it seems tricky to implement in this case because we still need the mysql service for grafana (and setup differs between the compose files), so it sounds like we would need one mysql profile besides a postgresql one, requiring to specify one or the other. I think I prefer to keep things simple for our default use case for now.